staging/src/k8s.io/apiserver/pkg/cel/environment/base.go

```diff
@@ -77,9 +77,10 @@ var baseOpts = []VersionedOptions{
 		IntroducedVersion: version.MajorMinor(1, 28),
 		EnvOptions: []cel.EnvOption{
 			cel.CrossTypeNumericComparisons(true),
-			// TODO: Add CEL Optionals once we bump cel-go
+			cel.OptionalTypes(),
```

Comment:
nit: do we want to add a TODO for enabling using optional next release/update default version?

Reply:
Nope, it's "automatic". We organized the base env with `IntroducedVersion` so we don't have to manually remember to turn things on.